src/modules/utils.js

```diff
@@ -258,4 +266,3 @@ function createUUIDPart(length) {
     }
```

Comment:
What do you think if in the near future we provide a built-in library for this? like loadash or underscore, and we keep our `utils` module only for kind of utility that are not available in those libraries.